packages/bpk-component-scrollable-calendar/src/BpkScrollableCalendarGridList.js

```diff
@@ -99,17 +99,6 @@ class BpkScrollableCalendarGridList extends React.Component {
         rowRenderer={this.rowRenderer}
         rowCount={this.state.months.length}
         overscanRowCount={0}
-        scrollToIndex={
```

Comment:
Removing this code as it was reintroduced during the reverting of the library being used for the scrollable calendar, and an issue that was raised via slack regarding the calendar jumping around when a date is selected, as its trying to scroll to indexes.

This will probably be raised as a separate PR or can be done in this cleanup?